kubernetes/README.md

```diff
@@ -25,7 +25,38 @@ Follow [this](https://docs.gitlab.com/ee/user/project/clusters/add_remove_cluste
 ```
 kubectl -n polkassembly create secret generic regcred --from-file=.dockerconfigjson=$HOME/.docker/config.json --type=kubernetes.io/dockerconfigjson
 kubectl -n polkassembly get secrets
-```  
+```
 to create the *secret* containing the required credentials in k8s.
 * make the credentials available from within gitlab-ci (required to push images to the registry).
 `cat $HOME/.docker/config.json | tr -d '\n'`. cut&paste resulting String into gitlab-ui -> Settings -> CI/CD -> Variables.
+
+4. Manual Steps needed after deployment to run polkassembly in production:
+
+- Create a bot user and set its id/credentials in auth server and chain db watcher
+  - create a user named Bot User through signup
+  - set botuser username in gitlab ci variable PROPOSAL_BOT_USERNAME. Needed as env var in chain db watcher.
```

Comment:
Variables are set already, we should check them. Env var are also set, so that's just an info.
```suggestion
  - verify the username in gitlab ci variable PROPOSAL_BOT_USERNAME. This is used as env var in chain db watcher.
```